react/week2/new-todo/src/Todolist.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+import Todoitem from "./Todoitem";
+
+function Todolist({ todos }) {
+  const [itemList, setItemList] = useState(todos);
+
+  function addNewItem() {
+    const newItem = {
+      id: Math.floor(Math.random() * 100),
+      description: "add task",
+    };
+    setItemList(itemList.concat(newItem));
```

Comment:
Use advance spread operator.
setItemList(...itemList,newItem);

Also try to add Input field to have different todo.